databricks-sdk-java/src/main/java/com/databricks/sdk/core/ProxyConfig.java

```diff
@@ -7,6 +7,8 @@ public class ProxyConfig {
   private String password;
   private ProxyAuthType proxyAuthType;
   private Boolean useSystemProperties;
+  // Needs the nonProxyHosts to be set in the format of "suffix1,suffix2,suffix3"
```

Comment:
Let's use | to separate hosts rather than , to follow the existing system property standard (https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html)